cogs/fun.py

```diff
@@ -25,6 +25,74 @@ def __init__(self, bot: Bot) -> None:
             "User-Agent": "HotWired",
             "Accept": "text/plain",
         }
+        self.session = aiohttp.ClientSession()
+
+    @command()
+    async def joke(self, ctx: Context) -> None:
+        """Sends a random joke."""
+        async with self.session.get("https://mrwinson.me/api/jokes/random") as resp:
+            if resp.status == 200:
+                data = await resp.json()
+                joke = data["joke"]
+                await ctx.send(
+                    embed=Embed(
+                        description=joke,
+                        color=Color.gold()
+                    )
+                )
+            else:
+                await ctx.send("Lmao! Something went boom! :(")
```

Comment:
We should include a real error message instead of this, that way we'll at least get the error code and it can be clearly reported, while this message is quite weird for that purpose.

Same goes for lines: 61, 78, 95